### DISCUSSION TITLE
Special Characters (encoding utf-8)

### DISCUSSION THREAD

**Discussion by qpqqq** [AntlerVC/firetable#1197] [Support / Q&A] [answered]
Hello , 
i have a CSV file full of data, some of this data contain German alphabet like (Ö, Ü, etc...). the CSV file shows all characters just file but when i import the table it's shows (? or some other symbol) so do you have any encoding extension in your website ?

**Comment by bofeiw** [comment, marked as answer]
It may be an encoding issue of import. We might need to implement alternative encoding to solve this issue

**Comment by qpqqq** [comment]
Yes, I think this is the most likely. I think there is a ready encoder built in typescript. Also I would like to thank you for your response.

**Comment by bofeiw** [comment]
@omarammura-dev can you share a sample import file to recreate this error? I dont know any German and can't recreate the issue. @bhaveek424 tried as well.
